mobile/src/main/java/org/openhab/habdroid/util/MyWebImage.java

```diff
@@ -128,8 +124,11 @@ public void onFailure(Call call, int statusCode, Headers headers, byte[] respons
             @Override
             public void onSuccess(Call call, int statusCode, Headers headers, byte[] responseBody) {
                 InputStream is = new ByteArrayInputStream(responseBody);
+                Map headersMap = headers.toMultimap();
+                boolean isSVG = headersMap.containsKey("content-type") &&
+                        headersMap.get("content-type").toString().contains("svg");
```

Comment:
This can be optimized by passing `response.body()` instead of `response.body().bytes()` to the callback and then using `responseBody.contentType()`. It's too invasive to do in this PR though; I'll prepare a follow-up one.